### DISCUSSION TITLE
멀티플레이 아바타의 복장만 교체할 수 있는지?

### DISCUSSION THREAD

**Discussion by SweetPotato7432** [naverz/zepeto-studio-kor#959] [5. Q&A]
안녕하세요, 제페토 월드 내부에 피팅룸과 같은 옷을 갈아입을 수 있는 기능 구현이 가능한지에 대해 문의 드립니다.
상호작용 가능한 물체를 터치 시 복장을 변경 시키는 기능을 구현하고 싶습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, https://github.com/naverz/zepeto-studio-kor/discussions/938 이슈에서 말씀드린 입어보기 기능이 제공되면 
구매 없이 단순 복장 교체도 가능해 집니다.
현재로서는 불가능하지만 곧 지원할 수 있도록 준비 중에 있습니다.
감사합니다.
